File: Python/1010.py
# Cálculo simples

produto1 = input()
produto2 = input()

lista1 = produto1.split()
lista2 = produto2.split()

total = (int(lista1[1]) * float(lista1[2])) + (int(lista2[1]) * float(lista2[2]))

print(f"VALOR A PAGAR: R$ {total:.2f}")

File: JavaScript/1010.js
// Cálculo simples

var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

let produto1 = lines.shift();
let produto2 = lines.shift();

let lista1 = produto1.split(" ");
let lista2 = produto2.split(" ");

let total =
  Number(lista1[1]) * Number(lista1[2]) + Number(lista2[1]) * Number(lista2[2]);

console.log(`VALOR A PAGAR: R$ ${total.toFixed(2)}`);
